docs/libcurl/opts/CURLOPT_SSL_CACHE_DATA.3

```diff
@@ -0,0 +1,86 @@
+.\" **************************************************************************
+.\" *                                  _   _ ____  _
+.\" *  Project                     ___| | | |  _ \| |
+.\" *                             / __| | | | |_) | |
+.\" *                            | (__| |_| |  _ <| |___
+.\" *                             \___|\___/|_| \_\_____|
+.\" *
+.\" * Copyright (C) 1998 - 2021, Daniel Stenberg, <daniel@haxx.se>, et al.
+.\" *
+.\" * This software is licensed as described in the file COPYING, which
+.\" * you should have received as part of this distribution. The terms
+.\" * are also available at https://curl.se/docs/copyright.html.
+.\" *
+.\" * You may opt to use, copy, modify, merge, publish, distribute and/or sell
+.\" * copies of the Software, and permit persons to whom the Software is
+.\" * furnished to do so, under the terms of the COPYING file.
+.\" *
+.\" * This software is distributed on an "AS IS" basis, WITHOUT WARRANTY OF ANY
+.\" * KIND, either express or implied.
+.\" *
+.\" **************************************************************************
+.\"
+.TH CURLOPT_SSL_CACHE_DATA 3 "22 Jul 2021" "libcurl 7.79.0" "curl_easy_setopt options"
+.SH NAME
+CURLOPT_SSL_CACHE_DATA \- custom pointer passed to ssl_cache callback
+.SH SYNOPSIS
+#include <curl/curl.h>
+
+CURLcode curl_easy_setopt(CURL *handle, CURLOPT_SSL_CACHE_DATA, void *pointer);
+.SH DESCRIPTION
+Data \fIpointer\fP to pass to the ssl cache callback set by the option
+\fICURLOPT_SSL_CACHE_FUNCTION(3)\fP, this is the pointer you'll get as fifth
+parameter.
+.SH DEFAULT
+NULL
+.SH PROTOCOLS
+All TLS based protocols: HTTPS, FTPS, IMAPS, POP3S, SMTPS etc.
+.SH EXAMPLE
+.nf
+#include <curl/curl.h>
+#include <stdio.h>
+
+static CURLcode sslcache_function(CURL *curl, struct ssl_session_dump **dump,
+                                  size_t *size, curl_free_callback *free_cb,
+                                  void *parm)
+{
+
+  /* all set to go */
```

Comment:
An example that doesn't populate any data in the callback isn't very useful is it?

Reply:
Right, and there are several other places where the docs could be improved. Just that I'll leave them last, just in case these changes will not be included in the lib.